static/src/javascripts/projects/common/modules/sport/score-board.js

```diff
@@ -44,7 +46,10 @@ define([
     ScoreBoard.prototype.componentClass = 'match-summary';
 
     ScoreBoard.prototype.prerender = function () {
-        this.placeholder.innerHTML = '';
+        var scoreLoadingPlaceholder = $('.score__loading', $(this.placeholder));
+        if (scoreLoadingPlaceholder.length) {
+            scoreLoadingPlaceholder.remove();
+        }
```

Comment:
When we first render, we want to remove the "Fetching the scores" before (`prerender`) we append the scores. We did this by setting `innerHTML`. However, this broke autoupdating, which happens after `prerender` and [expects the old element to be in the DOM for replacement](https://github.com/guardian/frontend/blob/1874c1d3873229751460687e3eebd627508e7555/static/src/javascripts/projects/common/modules/component.js#L246)